telegram/bot.py

```diff
@@ -1425,6 +1487,81 @@ def getChatMember(self, chat_id, user_id, **kwargs):
 
         return ChatMember.de_json(result, self)
 
+    def getWebhookInfo(self, **kwargs):
+        """Use this method to get current webhook status.
+
+        If the bot is using getUpdates, will return an object with the url field empty.
+
+        Returns:
+            :class: `telegram.WebhookInfo`
+
+        """
+        url = '{0}/getWebhookInfo'.format(self.base_url)
+
+        data = {}
+
+        result = self._request.post(url, data, timeout=kwargs.get('timeout'))
+
+        return WebhookInfo.de_json(result, self)
+
+    def setGameScore(self,
+                     user_id,
+                     score,
+                     chat_id=None,
+                     message_id=None,
+                     inline_message_id=None,
+                     edit_message=None,
+                     **kwargs):
+        """Use this method to set the score of the specified user in a game.
+
+        Returns:
+            :class:`telegram.Message`: The edited message.
+
+        """
+        url = '{0}/setGameScore'.format(self.base_url)
+
+        data = {'user_id': user_id, 'score': score}
+
+        if chat_id:
+            data['chat_id'] = chat_id
+        if message_id:
+            data['message_id'] = message_id
+        if inline_message_id:
+            data['inline_message_id'] = inline_message_id
+        if edit_message:
+            data['edit_message'] = edit_message
+
+        result = self._request.post(url, data, timeout=kwargs.get('timeout'))
+        return Message.de_json(result, self)
```

Comment:
`result` could be just `True` (https://core.telegram.org/bots/api#setgamescore), won't this raise an error?